### ISSUE TITLE
TrueType font support

### ISSUE THREAD

**Issue by mrichman** [Wandmalfarbe/pandoc-latex-template#68]
Is there any way to use a TrueType font? I suppose this will require XeLaTeX, but where would I specify the font name?

**Comment by daamien** [comment]
Have you tried setting the `fontfamily` variable ?

https://github.com/Wandmalfarbe/pandoc-latex-template/blob/master/eisvogel.tex#L101

**Comment by mrichman** [comment]
That doesn't work, even if I use `--pdf-engine=xelatex`, I get this error:

`! LaTeX Error: File 'Omnes.sty' not found.` 

where `Omnes` is the font family I specified with `fontfamily: Omnes` in the YAML header of my markdown file.

**Comment by Wandmalfarbe** [comment]
The `fontfamily` variable is only for fonts that are available as packages. Additional options (with the variable `fontfamilyoptions`) may be necessary for the font to show up in your document. It may be a bug in my template that custom fonts via `fontfamily` are overwritten with Source Code Pro... i'll have to investigate that.

TrueType system fonts are handled with XeTeX and [the options](https://pandoc.org/MANUAL.html#variables-for-latex) `mainfont`, `romanfont`, `sansfont`, `monofont` and `mathfont`. [This stackoverflow post](https://tex.stackexchange.com/a/417590) gives a nice overview and introduction on custom fonts with pandoc.

Basically you have to specify your ttf font in a YAML header and run pandoc with the XeTeX engine:

```yaml
---
mainfont: "My Custom Font"
...
```

**Comment by aggsol** [comment]
I would be great if we could move knowledge from closed issues like here into the wiki. It took some time yo find this issue while changing font is something most people want.
